romans_to_nums_solo.py

```diff
@@ -0,0 +1,48 @@
+""" Roman numerals are usually written largest to smallest from left to right.
+However, the numeral for four is not IIII. Instead, the number four is written
+as IV. Because the one is before the five we subtract it making four. The same
+principle applies to the number nine, which is written as IX. There are six
+instances where subtraction is used:
+
+I can be placed before V (5) and X (10) to make 4 and 9.
+X can be placed before L (50) and C (100) to make 40 and 90.
+C can be placed before D (500) and M (1000) to make 400 and 900.
+Given a roman numeral, convert it to an integer. """
+
+
+def romanToInt(input_string: str) -> int:
+	romans = {
```

Comment:
nit: indents look off, make sure its 4 spaces?

Reply:
The indents in sublime are set to 4 but the file looks different on here than on sublime